### PR TITLE
Resolved bottom Navbar Text wrap for all screens

### LINKED ISSUE #114: Text wrap in bottom Nav actions.
## Describe the bug

In the contents of the bottom nav, the text for the "My class" label gets wrapped for smaller android screens when the user switches between contents of the bottom nav, which leads icon for my class uplifted a bit higher (seems like overflowing)  as compared to other icons which sets a bad UI example.

## To Reproduce

1.swipe between the contents of the bottom nav and stay on my class label

currently on the web:
before-bug

bug-bottonNav

## Expected behavior

after-bug

icon for "my class" label should be even with other labels means behave same as other labels no text wrap 

## Environment (please complete the following information):

- OS: windows
- Browser: chrome

### PR DESCRIPTION
**Pull request by Abhijay007**
fixes #114

**Describe the changes you've made**

using CSS media query I resolved the text wrap bug in "MyClass" for the smallest screens possible (galaxy fold, I phone 5 ) in mobile view. 

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

How someone can test your code? 

using chrome inspect 

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/layouts/MainLayout/BottomNav.css [hunk 1]
```diff
@@ -2,10 +2,35 @@
   border-bottom: 2px solid #3949ab;
 }
 
-.MuiBottomNavigationAction-label.Mui-selected{
-  font-size: 0.8rem !important;
+.Mui-selected{
+  font-size: 0.8rem !important ;
+
```
Comment (rajatgupta24) [inline src/layouts/MainLayout/BottomNav.css:7]: Can you please remove this space.

**Review by rajatgupta24 — APPROVED** [review]
LGTM

**Comment by Abhijay007** [comment]
Ok

**Review by rajatgupta24 — APPROVED** [review]
LGTM but there is a error, check it first

**Pull request merged**